src/platform/posix/posix_resolv_gai.c

```diff
@@ -34,6 +34,10 @@
 // changed with this define.  Note that some platforms may not have a
 // thread-safe getaddrinfo().  In that case they should set this to 1.
 
+#ifdef NNG_RESOLV_CONCURRENCY
+#define NNG_POSIX_RESOLV_CONCURRENCY NNG_RESOLV_CONCURRENCY
+#endif
+
 #ifndef NNG_POSIX_RESOLV_CONCURRENCY
```

Comment:
My gut instinct is that this can be made slightly better.

Basically, I think we can eliminate the NNG_POSIX_RESOLV_CONCURRENCY (and also the Windows one below) and replace this with NNG_RESOLV_CONCURRENCY:

```
#ifndef NNG_RESOLV_CONCURRENCY
#define NNG_RESOLV_CONCURRENCY 4
#endif
```

Then the references to the old values should be replaced with this one new value.

It's a small change, but I think that will result in cleaner and clearer code, in case someone comes back here in the future.

Reply:
Basically what I mean is, we should just rename the old values to the new value.  Then you don't need the new block of code, because the old code already takes care of it.  Just need to update for your new unified name.

Reply:
Makes sense. I was trying not to change too much, but I agree it would be clearer to just replace the platform-specific macros. I'll make the change.